### PR TITLE
Add kubectl-hlf plugin

### PR DESCRIPTION
**Pull request by Dviejopomata**

### CODE REVIEW

**Comment by Dviejopomata** [comment]
I signed it

File: plugins/hlf.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hlf
+spec:
+  version: "v1.0.0"
+  homepage: https://github.com/kfsoftware/hlf-operator
+  shortDescription: "Deploy and manage Hyperledger Fabric Peers, Orderers and Certificate authorities"
```
Comment (ahmetb) [inline plugins/hlf.yaml:8]: Please try to shorten this to ~50 chars.

File: plugins/hlf.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  description: |
+    The kubectl-hlf plugin wraps the Hyperledger Fabric Operator and provides a simple
+    interface to create and manage Hyperledger Fabric Peers, Orderers and Certificate authorities
```
Comment (ahmetb) [inline plugins/hlf.yaml:11]: Please try to  wrap lines at 80 chars for readability in user's cli.

**Review by ahmetb — CHANGES_REQUESTED** [review]
Thanks for your submission.

**Comment by ahmetb** [comment]
You need to amend your commit to remove my handle.

Also, have you followed this advice about auth plugins: https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/#auth-plugins

**Comment by Dviejopomata** [comment]
No, I'm fixing it.
https://github.com/kfsoftware/hlf-operator/pull/6

I have already fixed the commit message.

**Comment by ahmetb** [comment]
/lgtm
/assign @corneliusweig 
For approval

**Comment by Dviejopomata** [comment]
Is there anything I can do from my side to merge this PR? @ahmetb

**Comment by corneliusweig** [comment]
Sorry it takes so long, but it's holiday season and that includes kids :).
Will take a look when possible. In the meantime thanks for your patience!

**Review by corneliusweig — COMMENTED** [review]
/approve
Thanks for contributing! This looks pretty interesting.

**Comment by ahmetb** [comment]
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.
